Remotion/Web/Development.WebTesting.UnitTests/Configuration/TestInfrastructureConfigurationTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Coypu;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.UnitTests.Configuration
+{
+  [TestFixture]
+  public class TestInfrastructureConfigurationTest
+  {
+    private readonly string _testRequestErrorDetectionStrategyAssemblyQualifiedName = typeof (TestRequestErrorDetectionStrategy).AssemblyQualifiedName;
+
+    private readonly MethodInfo _webTestConfigurationSectionIndexerSetter = typeof (WebTestConfigurationSection)

```

Comment:
Apparently, the name was wrong in the first place. It confused me so much that it made me not understand the code. This is much better and we don't need an additional method, you're right. We went from 😕 to 😁